src/southbridge/amd/pi/hudson/early_setup.c

```diff
@@ -255,4 +261,42 @@ void hudson_tpm_decode_spi(void)
 							| ROUTE_TPM_2_SPI);
 }
 
+/*
+ * Enable 4MB (LPC) ROM access at 0xFFC00000 - 0xFFFFFFFF.
```

Comment:
I think those references to 4MB are outdated, but I may be wrong. Perhaps they should be updated/removed, since this code is moved around anyway.

Reply:
```suggestion
 * Enable LPC ROM access at 0xFFXX0000 - 0xFFFFFFFF.
```

Would that suffice?

Reply:
Yes, but please also update comment inside the function, even if just by adding `...` or `etc.`

Reply:
https://github.com/Dasharo/coreboot/pull/467/files#r1523624918